src/lib.rs

```diff
@@ -488,6 +488,12 @@ impl fmt::Display for Level {
     }
 }
 
+impl From<LevelFilter> for Level {
+    fn from(level: LevelFilter) -> Self {
+        Level::from_usize(level as usize).unwrap()
```

Comment:
This is not a valid implementation - it will panic for LevelFilter::Off.

Reply:
You're right, I've removed it, I thought about using TryInto but it would also increase the minimum rust version with no need.